packages/web-components/README.md

```diff
@@ -0,0 +1,36 @@
+<div align="center">
+  <a href="https://prestashop.com">
+    <img src="https://prestashop.com/sites/default/files/email/logo_x2_rebrand.png" alt="Logo" width="420">
+  </a>
+
+<h3 align="center">Puik Web Components</h3>
+
+  <p align="center">
+    The <b>P</b>restaShop <b>UI</b> <b>K</b>it Web Components
+  </p>
+  <a href="https://uikit.prestashop.com/">Documentation</a>
+</div>
+
+## About The Project
+
+Puik Web Components is a library that aims to provide a complete set of reusable components based on the PrestaShop Design System for all the PrestaShop ecosystem.
+
+## Prerequisites
+
+- Node.js LTS is required.
+
+## Installation
+
+```sh
+# NPM
+$ npm install @prestashopcorp/puik-web-components --save
+
+# Yarn
+$ yarn add @prestashopcorp/puik-web-components
+
+# pnpm
+$ pnpm install @prestashopcorp/puik-web-components
+```
+### Web-Components Registration
+
+run  `initAllCe()` in your application.
```

Comment:
It could be great to add an explaination for on-demand imports
Using initAllCe isn't the best way to use the web components as this could create unused JS in apps and performance issues

Reply:
Done by @mattgoud